### PR TITLE
Refactoring cache to use Class [single-responsibiliy principle]

### PR DESCRIPTION
**Pull request by guilhermedelyra**

### CODE REVIEW

File: pydemic_ui/info.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import streamlit as st
 
 import mundi
-from pydemic import cache
+from pydemic import Cache
```
Comment (sammyzord) [inline pydemic_ui/info.py]: Please revert these changes

File: pydemic_ui/info.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def ttl_cache(
     if force_streamlit:
         return st.cache(ttl=ttl, **kwargs)(fn)
     elif force_joblib:
-        return cache.ttl_cache(key, timeout=ttl, **kwargs)(fn)
+        return Cache.TTL(key, timeout=ttl, **kwargs)(fn)
```
Comment (sammyzord) [inline pydemic_ui/info.py]: Please revert these changes

**Review by sammyzord — CHANGES_REQUESTED** [review]

File: pydemic_ui/info.py (hunk 2 above)
Comment (guilhermedelyra) [reply pydemic_ui/info.py]: Done!

File: pydemic_ui/info.py (hunk 1 above)
Comment (guilhermedelyra) [reply pydemic_ui/info.py]: Done

**Comment by guilhermedelyra** [comment]
Everything's working properly, I guess.

**Review by sammyzord — APPROVED** [review]

**Pull request merged**